src/components/ModelCamera/ModelCamera.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react'
+import dynamic from 'next/dynamic'
+import { Models } from "../../types/Models";
+import { loadModel } from './components/Model';
+
+
+const DynamicCamera = dynamic(() => import('./components/Camera'), {
```

Comment:
What does this do? I'm not familiar with it.

Reply:
It implements lazy loading of a component. This allows a component to NOT be SSR. In this case, we need the component to be CSR, since it uses tensorflow
https://nextjs.org/docs/advanced-features/dynamic-import